README.md

```diff
@@ -40,141 +40,141 @@ npm run lint-fix # format source files
-| [#208][#208] | [MUST define event payloads compliant with overall API guidelines][#208] | - |
-| [#209][#209] | [MUST maintain backwards compatibility for events][#209] | - |
-| [#210][#210] | [SHOULD avoid `additionalProperties` in event type definitions][#210] | - |
-| [#211][#211] | [MUST use unique event identifiers][#211] | - |
-| [#212][#212] | [SHOULD design for idempotent out-of-order processing][#212] | - |
-| [#213][#213] | [MUST follow naming convention for event type names][#213] | - |
-| [#214][#214] | [MUST prepare event consumers for duplicate events][#214] | - |
-| [#215][#215] | [MUST provide API identifiers][#215] | :heavy_check_mark: |
-| [#216][#216] | [SHOULD define maps using `additionalProperties`][#216] | - |
-| [#217][#217] | [MUST use full, absolute URI][#217] | - |
-| [#218][#218] | [MUST contain API meta information][#218] | :heavy_check_mark: |
-| [#219][#219] | [MUST provide API audience][#219] | - |
-| [#220][#220] | [MUST use most specific HTTP status codes][#220] | - |
-| [#223][#223] | [MUST-SHOULD use functional naming schema][#223] | - |
-| [#224][#224] | [MUST follow naming convention for hostnames][#224] | - |
-| [#225][#225] | [MUST follow naming convention for permissions (scopes)][#225] | - |
-| [#226][#226] | [MUST document implicit filtering][#226] | - |
-| [#227][#227] | [MUST document cachable `GET`, `HEAD`, and `POST` endpoints][#227] | - |
-| [#228][#228] | [MUST use URL-friendly resource identifiers: [a-zA-Z0-9:._\-/]*][#228] | - |
-| [#229][#229] | [SHOULD consider to design `POST` and `PATCH` idempotent][#229] | - |
-| [#230][#230] | [MAY consider to support `Idempotency-Key` header][#230] | - |
-| [#231][#231] | [Should use secondary key for idempotent `POST` design][#231] | - |
-| [#233][#233] | [MUST support `X-Flow-ID`][#233] | - |
-| [#234][#234] | [MUST only use durable and immutable remote references][#234] | - |
-| [#235][#235] | [SHOULD name date/time properties with `_at` suffix][#235] | - |
-| [#236][#236] | [SHOULD design simple query languages using query parameters][#236] | - |
-| [#237][#237] | [SHOULD design complex query languages using JSON][#237] | - |
-| [#238][#238] | [SHOULD use standardized property formats][#238] | - |
-| [#239][#239] | [SHOULD encode embedded binary data in `base64url`][#239] | - |
-| [#240][#240] | [SHOULD declare enum values using UPPER_SNAKE_CASE format][#240] | :heavy_check_mark: |
-| [#241][#241] | [MAY expose compound keys as resource identifiers][#241] | - |
+| id | title |supported | prio (A, B, C)|
+| --- | --- | --- | --- |
+| [#100][#100] | [MUST follow API first principle][#100] | - | B |
+| [#101][#101] | [MUST provide API specification using OpenAPI][#101] | - | A |
+| [#102][#102] | [SHOULD provide API user manual][#102] | - | C |
+| [#103][#103] | [MUST write APIs using U.S. English][#103] | - | A |
+| [#104][#104] | [MUST secure endpoints with OAuth 2.0][#104] | - | ? |
```

Comment:
```suggestion
| [#104][#104] | [MUST secure endpoints with OAuth 2.0][#104] | - | B |
```

Reply:
I'm not quite sure if we want to insert this rule. Maybe we want a sidecar which handles the security for the service. What do you think?

Reply:
IMO this should be part of the spec no matter where it's implemented. For the customer the service looks the same - sidecar or not.